client-js/app/elements/labrad-registry.html

```diff
@@ -6,40 +6,188 @@
-        <th>Key Value</th>
-      </thead>
-      <tbody>
-        <template is="dom-repeat" items="{{dirs}}">
-          <tr>
-            <td>
-              <a is="app-link" path="{{item.url}}" href="{{item.url}}">
-               <dir-item kind="folder">{{item.name}}</dir-item>
-              </a>
-            </td>
-            <td></td>
-          </tr>
-        </template>
-        <template is="dom-repeat" items="{{keys}}">
-          <tr>
-            <td><dir-item kind="key">{{item.name}}</dir-item></td>
-            <td><span class="paper-font-body1">{{item.value}}</span></td>
-          </tr>
-        </template>
-      </tbody>
-    </table>
+    <div id="reg-menu"> 
+    <section onclick="clickHandler(event)">
+      <menu-button data-dialog="doDelete" name="delete" icon-name="delete" class = "done"></menu-button>
+      <menu-button data-dialog="newFolderDialog" name="new folder" icon-name="folder-open" class = "done"></menu-button>
+      <menu-button data-dialog="copyDialog" name="copy" icon-name="content-copy" class = "done"></menu-button>
+      <menu-button data-dialog="renameDialog" name="rename" icon-name="code" class= "done"></menu-button>
+      <menu-button data-dialog="newKeyDialog" name="new key" icon-name="create"  class = "done"></menu-button>
+     </section>
     </div>
+    <div id="reg-area"> 
+      <table>
+          <tbody class="table-header" is="selectable-table"> 
+            <th></th>
+            <th>Key/Dir Name</th>
+            <th>Key Value</th>
+          </tbody>
+          <tbody is="selectable-table"  id="dirList" attr-for-selected="name" selected="{{selDir}}" on-tap="selectDir">       
+            <template is="dom-repeat" items="{{dirs}}">
+                <tr name="{{item.name}}">
+                  <td>
+                    <iron-icon icon = "more-vert"></iron-icon>
+                  </td>
+                  <td>
+                   <a is="app-link" path="{{item.url}}" href="{{item.url}}">
+                     <dir-item kind="folder">{{item.name}}</dir-item>
+                    </a>
+                  </td>
+                  <td></td>
+                </tr>
+            </template>
+          </tbody>
+          <tbody is="selectable-table" is="keyList" attr-for-selected="name"  selected="{{selKey}}" on-tap="selectKey"> 
+            <template is="dom-repeat" items="{{keys}}">
+              <tr name="{{item.name}}">
+                <td>
+                  <iron-icon icon = "more-vert"></iron-icon>
+                </td>
+                <td>
+                  <dir-item kind="key">{{item.name}}</dir-item>
+                </td>
+                <td>
+                  <form is="iron-form" id="{{item.name}}">
+                      <iron-a11y-keys keys="tab" on-keys-pressed="incrementSelector">
+                       <paper-input-container>
+                        <label>{{item.value}}</label>
+                        <input is="iron-input" name="{{item.name}}">
```

Comment:
the input here doesn't deal with multiline text when input gets large. maybe use [iron-autogrow-textarea](https://github.com/PolymerElements/iron-autogrow-textarea/blob/master/iron-autogrow-textarea.html) instead?